package.json

```diff
@@ -2,6 +2,11 @@
   "name": "flood",
   "version": "1.0.0",
   "private": false,
+  "license": "GPL-3.0-only",
```

Comment:
For those who ask themselves, `GPL-3.0-only` comes from here https://spdx.org/licenses/